docs/help_out.txt

```diff
@@ -139,4 +139,5 @@ This check is optional and will be skipped when set to 0. (default: 0) [$EIGENDA
    --storage.concurrent-write-routines value                                  Number of threads spun-up for async secondary storage insertions. (<=0) denotes single threaded insertions where (>0) indicates decoupled writes. (default: 0) [$EIGENDA_PROXY_STORAGE_CONCURRENT_WRITE_THREADS]
    --storage.dispersal-backend value                                          Target EigenDA backend version for blob dispersal (e.g. V1 or V2). (default: "V1") [$EIGENDA_PROXY_STORAGE_DISPERSAL_BACKEND]
    --storage.fallback-targets value [ --storage.fallback-targets value ]      List of read fallback targets to rollover to if cert can't be read from EigenDA. [$EIGENDA_PROXY_STORAGE_FALLBACK_TARGETS]
+   --storage.write-on-cache-miss                                              While doing a GET, write to the cache if a blob is not found in the cache but is found in EigenDA. (default: false) [$EIGENDA_PROXY_STORAGE_WRITE_ON_CACHE_MISS]
```

Comment:
iiuc the semantics of our current flags, `--storage.cache-targets` will cause the PUT routes (only) to write certs to the cache after having dispersed right? (same for fallback but we prety much have internal consensus that fallback is redundant and we'll get rid of it eventually)

So turning on `--storage.write-on-cache-miss` will also populate the cache during retrieval (GET) routes.

Think this makes sense. Another possibility for the flag name is `--storage.use-cache-for-retrievals` or something.

cc @ethenotethan

Reply:
```suggestion
   --storage.write-on-cache-miss                                              While doing a GET, write to the cache if a cert/blob pair is not found in the cache but is found in EigenDA. (default: false) [$EIGENDA_PROXY_STORAGE_WRITE_ON_CACHE_MISS]
```

Reply:
Also your msg says "in the cache", but what if there are multiple cache-targets turned on. Do you write to each? What about fallback targets?

Reply:
It's actually using the same mechanism that the Put uses, so technically speaking will write to all the caches and fallbacks. I could change it to "secondary storages"
"write to the secondary storages if a cert/blob pair is not found in the cache but is found in EigenDA. (default: false)"
It could eventually rewrite a blob in a fallback source, but as it is the same one, it shouldn't be a problem